### DISCUSSION TITLE
Cannot Deploy

### DISCUSSION THREAD

**Discussion by EvanClevenger** [community/community#27832] [Other Feature Feedback, Questions, & Ideas]
For some reason Ive been having the hardest time trying to deploy my link to GitHub pages.  I see in the actions that i get a  ERROR : Fatal :No url found for submodule path '' in .gitmodules.

**Comment by jfullstackdev** [comment]
do you have that submodule ? is it a fork ? if that is a public repo then it's better to send the link so we can take a closer look
